Document "OKTETO_BUILD_<service>_SHA" in favor of "OKTETO_BUILD_<serv… (#223)
okteto/docs: src/content/cloud/okteto-cli.mdx

- `BUILDKIT_HOST`: the address of the [Okteto Cloud buildkit instance](/docs/cloud/build/). You can use it to build your images with any CLI tool that supports Buildkit builds.

In order to refer to images in the `build` section of the Okteto Manifest, you can use the following environment variables:

> - `${OKTETO_BUILD_<service>_IMAGE}`: the full image reference (using its SHA to force redeploy)

Reviewer: Can you add an example?

- `${OKTETO_BUILD_<service>_REGISTRY}`: the registry URL where the image was pushed

> - `${OKTETO_BUILD_<service>_REPOSITORY}`: the name of the image that was pushed (`namespace/_<service>`)

Reviewer: Is the underscore part of the value? (_service)

Author: fixed

> - `${OKTETO_BUILD_<service>_SHA}`: tag SHA for the last image build at the registry

Reviewer: What do you mean with "TAG sha"?

Reviewer: Add an example as well?

Author: done

- `BUILDKIT_HOST`: the address of the [Okteto Cloud buildkit instance](/docs/cloud/build/). You can use it to build your images with any CLI tool that supports Buildkit builds.

In order to refer to images in the `build` section of the Okteto Manifest, you can use the following environment variables:
- `${OKTETO_BUILD_<service>_IMAGE}`: the full image reference
- `${OKTETO_BUILD_<service>_REGISTRY}`: the registry URL where the image was pushed
- `${OKTETO_BUILD_<service>_REPOSITORY}`: the name of the image that was pushed

> - `${OKTETO_BUILD_<service>_SHA}`: tag SHA for the last image build at the registry

Reviewer: Is this what you mean?

```suggestion
- `${OKTETO_BUILD_<service>_SHA}`: The latest tag and the SHA of the image
```
